src/main/java/Entity/Message.java

```diff
@@ -7,6 +7,33 @@ public class Message {
     private String startTime;
     private String endTime;
 
+    //todo
+    public Message(String message) {
+        this.message = message;
+        this.isDone = false;
+        this.startTime = "";
+        this.endTime = "";
+        this.type = 1;
+    }
+
+    //event
+    public Message(String message, String startTime, String endTime, int type) {
```

Comment:
Maybe the use and reason for overloading the Message function could be made more explicit?